### ISSUE TITLE
Feat: Auto updating messages

### ISSUE THREAD

**Issue by SamNuttall** [SamNuttall/TruckersMP-Info-Bot#4]
Add an automatically updating message to a server channel, will be known as pins.
Will support server overview, highest traffic areas and all event categories (featured, upcoming, now).

**Comment by SamNuttall** [comment]
@MTCVTC Hey! This feature has rolled out and is currently being tested. Assuming you have the correct permissions in the Discord server, you should be able to use the `/pin` command.

**Comment by MTCVTC** [comment]
/pin available in open source project to?

**Comment by SamNuttall** [comment]
@MTCVTC Yes, the feature is currently avaliable on the feat-auto-msgs branch.

**Comment by MTCVTC** [comment]
[not showing emojis in any slash commmads ](https://cdn.discordapp.com/attachments/1100380334362738719/1125283114705498292/IMG_20230703_100300.jpg)


am using open source project

**Comment by SamNuttall** [comment]
@MTCVTC Alfie uses custom emojis which are available in one of its servers, hence why the bot can use them elsewhere. When hosting the bot manually, these emojis cannot be accessed. I recommend using the official bot to overcome this.

If you insist on self hosting your own solution, the emoji required are provided in the source code and you can edit the emoji configuration [here](https://github.com/SamNuttall/TruckersMP-Info-Bot/blob/main/common/ui/emoji.py). Note that this file does contain a `USE_EXCLUSIVE` mode, which would disable the use of custom emojis but does not do anything at this time.

**Comment by MTCVTC** [comment]
main branch is working nice but feat-auto msgs branch have some errors that i can't make it right 

 File "/home/container/main.py", line 19, in <module>
    main()
  File "/home/container/main.py", line 14, in main
    startup_utils.load_exts(bot)
  File "/home/container/common/startup_utils.py", line 70, in load_exts
    bot.load_extension(ext)
  File "/home/container/.local/lib/python3.11/site-packages/interactions/client/client.py", line 1995, in load_extension
    module = importlib.import_module(module_name, package)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/container/exts/pins.py", line 13, in <module>
    from common.data.db import Guild
ImportError: cannot import name 'Guild' from 'common.data.db' (unknown location)

**Comment by MTCVTC** [comment]
POST::https://discord.com/api/v10/channels/1125295781314822184/messages: 400
Ignoring exception in cmd `/pins add`: HTTPException: 400|Bad Request || Cannot send an empty message

this error comes using /pins add

**Comment by SamNuttall** [comment]
@MTCVTC I won't be providing detailed support for self hosting like I would if you were using the hosted version. 

However, please make sure you are using the latest version of the branch. If this still happens and you believe this is a genuine bug after doing this, please create a new issue and I'll take a look at it.
